packages/server/src/parse-chokidar-rules-from-gitignore.ts

```diff
@@ -0,0 +1,55 @@
+import parseGitignore from 'parse-gitignore'
+import fs from 'fs'
+import {resolve} from 'path'
+import _ from 'lodash'
```

Comment:
Would we get a tny install bundle optimization by using a destructured import? 

```ts
import {partition} from 'lodash'
```

Reply:
I don't know, do we have tree shaking working? I generally prefer having the `_` there, since it makes it obvious we're using lodash (familiar to a lot of people).

Reply:
Also I can't see we're using lodash anywhere else as it is probably brought in by a sub dependency. If we are using explicitly I would probably suggest we explicitly add it.